examples/docker/README.md

```diff
@@ -0,0 +1,44 @@
+# Example docker image for theia-trace-extension
+
+This folder contains an example showing how to create a docker image
+for theia-trace-ext front-end.
+
+Note that the image will contain exclusively the theia-trace-ext
+front-end. If you want to run a complete application, you will need
+a service running the trace-server (not included here).
+
+Note also that the image will be built using the latest [npm package]
+of the theia-trace-extension, and not the code in this repo.
+
+# How to build and run
+
+Build the image and name it *tte*:
+```
+docker build -t tte .
+```
+
+Once the image has been built, start a container named *tte-1* from
+the *tte* image:
+```
+docker run --name tte-1 tte
+```
+
+Find the IP address of the *tte-1* container:
+```
+docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' tte-1
+```
+
+Connect to port 4000 of the IP identified earlier using your browser.
+You should be able to see the theia-trace-ext UI.
+
+# Connect to a trace-server
+
+Let's say you have anoher container running the trace-server at
+IP 172.17.0.2, port 8080. You can launch the *tte-1* container and
+connect it to the trace-server using the following command:
+
+```
+docker run --name tte-1 --network="host" -e TRACE_SERVER_URL=172.17.0.2:8080/tsp/api tte
```

Comment:
If I run a trace-server locally as `http://localhost:8080/tsp/api`, non-containerized but say from an Eclipse debug config, then I'm unable to get line 28's IP. I rather get an empty output for that command. I tried with a few variations of the command right above (line 41's), without success. But WDYT?

Reply:
Nevermind, the viewer IP is `localhost`('s) in this case. And the container doesn't reveal that since trivial (or per `host` network definition) in its `inspect` output. Maybe clarify this in the README, around here or so?